src/script/audacity-pipe.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Audacity pipe.
+Keep audacity-pipe.py short!!
+You can make more complicated longer tests to test other functionality
+or to generate screenshots etc in other scripts.
+Make sure Audacity is running first and that mod-script-pipe is enabled
+before running this script.
+Requires Python 2.7 or later. Python 3 is strongly recommended.
+"""
+
+import os
+import sys
+import time
+import re
+
+if (len(sys.argv) < 2):
+    print("Usage: python audacity-pipe.py <fullpath>")
+    sys.exit(-1)
+prefs = os.path.join('C:\\Users\\Trihus\\AppData\\Roaming', 'audacity', 'audacity.cfg')
```

Comment:
what are we doing about this?

Reply:
So in theory, this code wouldn't run because you won't get her without enabling scripting in the calling program but I put this hear when testing the script from the command line. Originally I use os.getenv('AppData') instead of the hard path and that will probably work again now. (This code isn't actually running with this check in because I removed the icon on the data sheet to launch the python script. I think we decided we wanted to do more thinking about the process of defining and launching audacity. For now, I'll just replace the absolute path.